utils/frame/remote-externalities/src/lib.rs

```diff
@@ -501,16 +505,53 @@ impl<B: BlockT> Builder<B> {
 	pub async fn build(self) -> Result<TestExternalities, &'static str> {
 		let kv = self.pre_build().await?;
 		let mut ext = TestExternalities::new_empty();
+		let (offchain, _offchain_state) = TestOffchainExt::new();
+
+		// Register externality extensions in order to provide host interface for OCW to the runtime
+		ext.register_extension(OffchainDbExt::new(offchain.clone()));
```

Comment:
no -- we probably don't want to do this here. the call site can take the `ext` and add the extensions to it. We want to do this in the `offchain_worker` command of `try-runtime-cli` stuff.